src/video/VDP.hh

```diff
@@ -372,6 +372,31 @@ public:
 		       (!blinkState << 8);
 	}
 
+	/** Expresses the state of even/odd page interchange in a mask
+	  * on the line number. If even/odd interchange is active, for some
+	  * frames lines 256..511 (page 1) are replaced by 0..255 (page 0)
+	  * and 768..1023 (page 3, if appicable) by 512..767 (page 2).
```

Comment:
applicable

Reply:
Last commit fixes this and the original comment